src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -101,6 +105,9 @@ export const SiteDashboard = (): JSX.Element => {
     ({ status }) => status === "OPEN" || status === "APPROVED"
   )
 
+  const { siteLaunchStatus } = siteLaunchStatusProps
+  // todo add loading state for site launch api call, after IS-219
```

Comment:
is there a tracking issue or is this near term? if it's near term then omitting the issue is ok i feel

Reply:
removed liao